src/components/News.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import Card from "./Card";
+import { getAllArticles } from "../api";
+
+const News = () => {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    getAllArticles().then((result) => setArticles(result.articles));
+  }, []);
+  console.log(articles);
```

Comment:
You could include a loading state variable here and use it to conditionally render something that indicates to a user that a background process is ongoing, this goes a long way for users who are on a slower connection

There is also a floating console log in there 🧹